wagtail/admin/templates/wagtailadmin/workflows/edit_task.html

```diff
@@ -41,8 +41,16 @@
                     <li class="actions footer__container">
                         {% block form_actions %}
                             <div class="dropdown dropup dropdown-button match-width">
-                                <button type="submit" class="button action-save button-longrunning" data-clicked-text="{% trans 'Saving…' %}">
-                                    {% icon name="spinner" %}<em>{% trans 'Save' %}</em>
+                                <button 
+                                    type="submit" 
+                                    class="button action-save button-longrunning" 
+                                    data-action="w-loader#activate"
+                                    data-controller="w-loader"
+                                    data-w-loader-active-class="button-longrunning-active"
+                                    data-w-loader-active-value="{% trans 'Saving…' %}"
+                                    data-w-loader-duration-seconds-value="40"
```

Comment:
Just a quick passing thought. We should see if Stimulus supports numeric separators.

https://2ality.com/2018/02/numeric-separators.html

This way we could write as this (means we can use milliseconds consistently but also preserve readability) - have a check and see if it works.

Note: this is an ES2021 feature so not sure, plus not sure how the number parsing works in Stimulus but worth raising an issue in that repo if not supported I think.

```suggestion
                                    data-w-loader-duration-value="40_000"
```

Reply:
I tried this. The timeout still works but when i console the `this.durationSecondsValue`, I get `NaN`
So I will skip the implemetation of this for now
Couldn't find an article on this related to Stimulus

Reply:
No problems. I have investigated the Stimulus code and see it does not support this numeric separators syntax. 

I have raised an issue on their repo - let's see if they are keen to add it. If so, we will use milliseconds instead for consistency. 

https://github.com/hotwired/stimulus/issues/646